### PR TITLE
Open pull request feature

### PR DESCRIPTION
**Pull request by nathanbaleeta**
Added feature for opening pull request using new DPG submission form.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,102 +1,155 @@
...
+  const TOKEN = process.env.ACCESS_TOKEN; // create token at https://github.com/settings/tokens/new?scopes=repo
+  const octokit = new MyOctokit({
+    auth: TOKEN,
+  });
+
+  // Returns a normal Octokit PR response
+  // See https://octokit.github.io/rest.js/#octokit-routes-pulls-create
+  octokit
+    .createPullRequest({
+      owner: "nathanbaleeta",
```
Comment (lacabra) [inline pages/index.js:105]: Let's make these 2 variables at the top of the file

File: pages/index.js [hunk 2]
```diff
@@ -1,102 +1,155 @@
...
+      repo: "submission-form",
+      title: `Add nominee: ${values.name}`,
+      body:
+        "Automatic addition of a new nominee submitted through the online form available at https://digitalpublicgoods.net/submission",
+      base: "main" /* optional: defaults to default branch */,
```
Comment (lacabra) [inline pages/index.js:110]: Make this a variable as well.

**Review by lacabra — CHANGES_REQUESTED** [review]
Looking good so far @nathanbaleeta, just a couple of minor requests while you are still working on it. Thanks!

File: pages/index.js (hunk 1 above)
Comment (nathanbaleeta) [reply pages/index.js]: Resolved in commit 225d4f0.

File: pages/index.js (hunk 2 above)
Comment (nathanbaleeta) [reply pages/index.js]: Resolved in commit 225d4f0.

File: pages/index.js [hunk 3]
```diff
@@ -1,102 +1,174 @@
...
     }
   }
 
   return errors;
+};
+
+function openPR(values) {
+  let sdgNumber, evidenceText;
+  for (let i = 0; i < values.SDGs.length; i++) {
+    sdgNumber = JSON.stringify(values.SDGs[i]);
```
Comment (lacabra) [inline pages/index.js:97]: Why is this needed? couldn't you combine L97 and L98 into one as follows?
```
sdgNumber = parseInt(values.SDGs[i])
```

File: pages/index.js [hunk 4]
```diff
@@ -1,102 +1,174 @@
...
+      evidenceText: values[evidenceText],
+    };
+
+    delete values[evidenceText];
+  }
+
+  const myJSON = JSON.stringify(values, null, "\t");
+
+  // Return project json file aligned with naming convention
+  let name = values.name.toLowerCase().replace(/ /g, "_") + ".json";
```
Comment (lacabra) [inline pages/index.js:112]: I know that you have taken this from other existing files, but this is not enough. We need to replace characters like `á` for `a` and the like.

This line written in bash has the expanded replacement that we would need to port to Javascript. Would you mind taking care of this?

https://github.com/unicef/publicgoods-candidates/blob/ecd742361eb42e74208948d111d00c473ec22bcc/scripts/check-filenames.bash#L14

File: pages/index.js [hunk 5]
```diff
@@ -1,102 +1,174 @@
...
+  octokit
+    .createPullRequest({
+      owner: GITHUB_OWNER,
+      repo: GITHUB_REPO,
+      title: `Add nominee: ${values.name}`,
+      body:
+        "Automatic addition of a new nominee submitted through the online form available at https://digitalpublicgoods.net/submission",
+      base: GITHUB_BRANCH,
+      head:
+        `${values.name}`.replace(/ /g, "_") +
```
Comment (lacabra) [inline pages/index.js:131]: For uniformity, I suggest we reuse the processed 'name' from L112 above (without the `.json` extension obviously)

**Review by lacabra — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 3 above)
Comment (nathanbaleeta) [reply pages/index.js]: Resolved in commit 8988263.

File: pages/index.js (hunk 5 above)
Comment (nathanbaleeta) [reply pages/index.js]: Resolved in commit a7c7df8.

File: pages/index.js (hunk 4 above)
Comment (nathanbaleeta) [reply pages/index.js]: Resolved in commit 5740843.

File: pages/index.js [hunk 6]
```diff
@@ -127,7 +127,7 @@ function openPR(values) {
         "Automatic addition of a new nominee submitted through the online form available at https://digitalpublicgoods.net/submission",
       base: GITHUB_BRANCH,
       head:
-        `${values.name}`.replace(/ /g, "_") +
+        `${name}`.split(".").slice(0, -1).join(".") +
```
Comment (lacabra) [inline pages/index.js:135]: Very elegant 🙃

File: pages/index.js [hunk 7]
```diff
@@ -107,8 +107,13 @@ function openPR(values) {
 
   const myJSON = JSON.stringify(values, null, "\t");
 
-  // Return project json file aligned with naming convention
-  let name = values.name.toLowerCase().replace(/ /g, "_") + ".json";
+  // Return project json file aligned with naming convention(includes removing accents)
+  let name =
+    values.name
+      .normalize("NFD")
```
Comment (lacabra) [inline pages/index.js:113]: Very cool! 🎉

**Review by lacabra — APPROVED** [review]
